SPECS/k3s/k3s-1.25.8.spec

```diff
@@ -82,11 +82,8 @@ exit 0
 %{install_sh}
 
 %changelog
-* Tue Apr 11 2023 Anuj Garg <anujgarg@microsoft.com> - 1.25.5-2
-- Addressed CVE for containerd and changed it to the new patched version 1.6.18
-
-* Wed Apr 05 2023 Anuj Garg <anujgarg@microsoft.com> - 1.25.5-1
-- Updated k3s to 1.25.5
```

Comment:
Same question as with `k3s.spec`

Reply:
Dropped these change logs because they have been replaced with the new version changes so the older change logs will not be valid that's why removed.

Reply:
Ah, the other versions there haven't been published, I see. That's fine then.